fe/fe-core/src/main/java/org/apache/doris/nereids/stats/FilterEstimation.java

```diff
@@ -17,7 +17,9 @@
 
 package org.apache.doris.nereids.stats;
 
+import org.apache.doris.analysis.DateLiteral;
```

Comment:
why use legacy planner's DateLiteral? because ColumnStatistic' expr is legacy literal? i think we need to update Expr in ColumnStatistic to Nereids' literal in future

Reply:
yes, ColumnStatistic' expr is legacy literal